apis/outlook/calendar/code/main.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/gptscript-ai/tools/apis/outlook/calendar/code/pkg/commands"
+	"github.com/gptscript-ai/tools/apis/outlook/calendar/code/pkg/graph"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Println("Usage: calendar <command>")
+		os.Exit(1)
+	}
+
+	command := os.Args[1]
+
+	switch command {
+	case "listCalendars":
+		if err := commands.ListCalendars(context.Background()); err != nil {
```

Comment:
Right now I'm just using context.Background() everywhere, but this can be easily changed. Just not sure if it is something that needs to be changed.

Reply:
This sounds good to me.